packages/lib/CalendarManager.ts

```diff
@@ -169,7 +169,7 @@ export const getBusyCalendarTimes = async (
     // Subtract 11 hours from the start date to avoid problems in UTC- time zones.
     const startDate = dayjs(dateFrom).subtract(11, "hours").format();
     // Add 14 hours from the start date to avoid problems in UTC+ time zones.
-    const endDate = dayjs(dateTo).endOf("month").add(14, "hours").format();
+    const endDate = dayjs(dateTo).add(14, "hours").format();
```

Comment:
This was messing with cache so bad. We also do end of month on frontend so we ended up 4 months in the future. Also removing this shouldn't mess with regular logic.

Reply:
We already do endOf month in the front end BTW: https://github.com/calcom/cal.com/blob/fix/calendar-cache-parsing/packages/features/schedules/lib/use-schedule/useTimesForSchedule.ts#L38